animals_and_tips.html

```diff
@@ -18,7 +18,7 @@
 	<!--Navigation-->
 	<div class="Nav_Menu">
 	<div class="logo">
-		<a href="index.html"><img src="images/logo.png" height="40" alt="logo"></a>
+		<a href="index.html"><img src="images/home.png" height="40" alt="logo"></a>

```

Comment:
V7 logo was nice however i do like V8s home icon, as it was easier to recognize than the logo

Reply:
After looking back at it I think the logo for Version 7 is the best I will most likely choose this image for the design, however I will edit it to remove black outline